slider/slider.js

```diff
@@ -1,4 +1,160 @@
-let btn = document.querySelector("button");
+let btn = document.getElementById("button");
+var smallScreen = window.matchMedia("(max-width: 650px)")
+
+function genTable() {
+  let table = ``;
+  let numbers = sliderRange();
+  for (var row = 0; row < 11; row++) {
+    table += `<tr>`;
+    table += `<td>${numbers[row]}</td>`;
+    table += `<td>-</td>`;
+    table += `</tr>`;
+  }
+  document.getElementById("grid").innerHTML = table;
+}
+
+function sliderRange() {
+  const numbers = [];
+  for (var i = 0; i <= 100; i += 10) {
+    numbers.push(i);
+  }
+  return numbers.reverse();
+}
+
+function applySliderConfigs({ color, size, scale, defaultPosition }) {
+ 
+    btn.style.backgroundColor = color;
+
+    btn.style.height = size;
+
+  
+
+    btn.innerHTML = scale;
+
+
+
+    btn.style.top = defaultPosition;
+  
+};
+smallScreen.matches
+  ? applySliderConfigs({ color: "red", size: "50px", scale: 0, defaultPosition: "0px"})
+  : applySliderConfigs({ color: "green", size: "150px", scale: 0,  defaultPosition: "725px"})
+
+function buttonMove() {
+  let start = null;
+  btn.onmousedown = function (e) {
+    start = e.clientY;
+  };
+  smallScreen.matches
+    ? btn.onmousemove = function (e) {
```

Comment:
I mentioned in a previous comment that you can simplify this function in the if / else. can you think of a way to do that?